src/scs/scs.md

```diff
@@ -1,3 +1,7 @@
 # SCS
 
+Supply Chain Security is a process of securitng the machinery of the development, build and release environment, in other words, securing every component that a software artifact might be touching on its way from the developer towards the consumer. The software artifact should be encrypted on each possilbe transition phase and the integrity of it should be verified at each destination.
```

Comment:
And here, Supply Chain Security (SCS).
After that, you can use SCS to your heart's extent :-)

Reply:
I will add this information to our Style Guide.